openapi/consumed/api_remote_content.yaml

```diff
@@ -0,0 +1,524 @@
+        in: header
+        description: The url of the endpoint called by IO app
+        schema:
+          $ref: "#/components/schemas/LollipopOriginalURL"
+      - name: signature-input
+        in: header
+        description: The signature input, needed to verify the `signature` header
+        schema:
+          $ref: "#/components/schemas/LollipopSignatureInput"
+      - name: signature
+        in: header
+        description: The signature of the HTTP request, signed by the client with its private key.
+        schema:
+          $ref: "#/components/schemas/LollipopSignature"
+      - name: x-pagopa-lollipop-assertion-ref
+        in: header
+        schema:
+          $ref: "#/components/schemas/LollipopAssertionRef"
+      - name: x-pagopa-lollipop-assertion-type
+        in: header
+        schema:
+          $ref: "#/components/schemas/AssertionType"
+      - name: x-pagopa-lollipop-auth-jwt
+        in: header
+        schema:
+          $ref: "#/components/schemas/LollipopJWTAuthorization"
+      - name: x-pagopa-lollipop-public-key
+        in: header
+        schema:
+          $ref: "#/components/schemas/LollipopPublicKey"
+      - name: x-pagopa-lollipop-user-id
+        in: header
+        schema:
+          $ref: "#/components/schemas/LollipopUserId"
+
+
+      responses:
+        200:
+          description: OK
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/PreconditionContent'
+        400:
+          description: Bad Request
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        401:
+          description: Unauthorized
+          content: {}
+        403:
+          description: Forbidden
+          content: {}
+        404:
+          description: No message or precondition found for the provided ID.
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        429:
+          description: Too Many Requests
+          content: {}
+        500:
+          description: Internal Server Error
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+components:
+  schemas:
+    PreconditionContent:
+      required:
+      - markdown
+      - title
+      type: object
+      properties:
+        title:
+          type: string
+          description: The title to be rendered in App
+        markdown:
+          type: string
+          description: The markdown content to be rendered in App
+    RemoteContentMessage:
+      type: object
+      properties:
+        attachments:
+          type: array
+          items:
+            $ref: '#/components/schemas/RemoteContentAttachment'
+        details:
+          type: object
+          properties:
+            markdown:
+              $ref: "#/components/schemas/Markdown"
```

Comment:
I'd rather have extracted the whole "details" as a component, than the single Markdown and Subject.

Reply:
[9ad5ee0](https://github.com/pagopa/io-backend/pull/1077/commits/9ad5ee01580411047d44ffec0b64d41fe6a84284)